.travis.yml

```diff
@@ -1,2 +1,41 @@
+os: linux
+
+dist: bionic
+
+language: python
+
+Python:
+  - "3.7"
+
+before_install:
+  # require for installing pyodbc
+  - sudo apt-get install unixodbc-dev
+
+# command to setup datamegh
+install:
+  - make setup
+
+# check for black format
+# build will stop if check fail 
+before_script:
+  - black --check --diff . 
+
```

Comment:
These shouldn't be needed anymore as our tests run inside a container which has it all setup.